packages/legend-extension-application-studio-query-builder/src/components/QueryBuilder_LegendStudioApplicationPlugin.tsx

```diff
@@ -329,14 +329,14 @@ export class QueryBuilder_LegendStudioApplicationPlugin
                     }),
                   );
                   if (queryBuilderExtension.mode) {
-                    queryBuilderExtension.queryBuilderState.changeClass(
-                      classView.class.value,
-                    );
                     queryBuilderExtension.queryBuilderState.initialize(
                       diagramEditorState.editorStore.graphManagerState.graphManager.createGetAllRawLambda(
                         classView.class.value,
                       ),
                     );
+                    queryBuilderExtension.queryBuilderState.changeClass(
+                      classView.class.value,
+                    );
```

Comment:
@akphi this will fix the issue. But do we want to add `createGetAllRawLambda(` templates for `milestoning` too?

Reply:
no, I think for now, let's keep them simple